src/Slider/Slider.tsx

```diff
@@ -50,29 +56,50 @@ const Slider: React.FunctionComponent<SliderProps> = (props: SliderProps): React
             });
             setLabelsPositions(positions);
         }
-    }, [props.labels]);
+    }, [props.labels, min, max]);
+
+    React.useEffect(() => setThumbPosition(getPercentage()), [props.value, min, max, size]);
+
+    /**
+     * Finds the size between two numbers
+     * @param {number} minValue The minimum value
+     * @param {number} maxValue The maximum value
+     * @returns {number} The size
+     */
+    function getSize(minValue: number, maxValue: number): number {
+        if (maxValue > minValue) {
+            return maxValue - minValue;
+        } else { // Will calculate the size anyway, but it will show a warning since the min is larger than the max
+            console.warn(`The max value of the slider should be larger than the min value (Max:${max}, Min: ${min}`);
+            return minValue - maxValue;
+        }
+    }
 
     /**
      * Converts the current value to percentage based on min and max
-     * @param {number} value value to be converted to percentage
      * @returns {number} The precentage
      */
-    function getPercentage(value: number): number {
-        if (value < min) {
+    function getPercentage(): number {
+        if (props.value <= min) {
             return 0;
-        } else if (value > max) {
+        } else if (props.value >= max) {
             return 100;
         } else {
-            const offset: number = Math.abs(0 - min);
-            const result: number = Math.abs(((value - min) / (max - offset)) * 100);
-            return result;
+            const distanceFromMin: number = Math.abs(props.value - min);
+            // Calculating the percentage using this equation (distanceFromMin/size) * 100
```

Comment:
This comment is not necessary

Reply:
It's just to explain what's included in the calculation. Furthermore, I don't think this is worthy of a commit. Let's just leave it there.